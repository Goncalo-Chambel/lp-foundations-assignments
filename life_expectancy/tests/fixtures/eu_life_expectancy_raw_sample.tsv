unit,sex,age,geo\time	2021 	2020 	2019 	2018 	2017 	2016 	2015 	2014 	2013 	2012 	2011 	2010 	2009 	2008 	2007 	2006 	2005 	2004 	2003 	2002 	2001 	2000 	1999 	1998 	1997 	1996 	1995 	1994 	1993 	1992 	1991 	1990 	1989 	1988 	1987 	1986 	1985 	1984 	1983 	1982 	1981 	1980 	1979 	1978 	1977 	1976 	1975 	1974 	1973 	1972 	1971 	1970 	1969 	1968 	1967 	1966 	1965 	1964 	1963 	1962 	1961 	1960 
YR,F,Y32,FR	: 	53.9 p	54.6 p	54.5 p	54.4 	54.4 	54.2 	54.7 b	54.3 	54.0 b	54.3 	53.9 	53.7 	53.5 	53.5 	53.2 	52.5 	52.6 	51.5 	51.8 	51.8 	51.8 	51.6 	51.5 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 
YR,T,Y4,IS	: 	79.3 	79.3 	79.1 	78.9 	78.3 	78.8 	79.1 	78.3 	79.1 	78.5 	78.2 	78.0 	77.9 	77.8 	77.4 	77.8 	77.3 	77.3 	76.9 	77.0 	76.0 	75.6 	76.0 	75.4 	75.2 	74.6 	75.6 	75.4 	75.3 	74.5 	74.6 	74.6 	73.7 	74.0 	74.4 	74.2 	74.1 	73.3 	73.7 	73.0 	73.5 	: 	73.7 	73.2 	73.7 	72.7 	71.4 	71.4 	71.4 	70.7 	71.0 	70.8 	71.2 	70.9 	70.5 	71.3 	71.1 	70.6 	71.3 	71.1 	: 
YR,M,Y19,RU	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	45.1 	44.9 	44.0 	43.7 	42.7 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 
YR,T,Y_GE85,PT	: 	6.2 	6.7 	6.5 	6.5 	6.4 	6.3 	6.5 	6.4 	6.2 	6.7 	6.1 	6.1 	6.0 	6.0 	6.1 	5.6 	6.0 	5.4 	5.7 	5.8 	5.6 	5.1 	5.1 	5.1 	4.8 	4.8 	4.9 	4.5 	4.9 	4.6 	4.5 	4.8 	4.5 	4.5 	4.4 	4.1 	4.1 	4.1 	4.5 	4.4 	4.2 	4.1 	3.7 	3.9 	3.6 	3.7 	3.5 	3.9 	4.2 	4.0 	3.9 	3.1 	3.3 	3.3 	3.4 	3.7 	3.9 	4.1 	4.7 	4.8 	4.2 
YR,F,Y33,EE	: 	50.4 	50.6 	50.2 	50.3 	50.1 	50.0 b	49.7 	49.4 	49.4 	49.1 	48.5 	48.1 	47.6 	47.0 	46.6 	46.4 	46.1 	45.4 	45.2 	45.0 	44.8 	44.7 	44.0 	44.4 	44.1 	43.4 	42.4 	43.1 	43.9 	43.7 	43.8 	44.1 	44.1 	44.2 	44.2 	43.4 	43.4 	43.9 	43.9 	43.5 	43.7 	43.5 	43.6 	43.7 	43.7 	43.9 	44.5 	44.0 	43.9 	44.1 	43.7 	43.7 	44.1 	44.4 	43.7 	43.8 	44.2 	43.6 	43.2 	43.4 	43.6 
YR,T,Y16,PT	: 	65.4 	66.2 	65.9 	65.9 	65.7 	65.7 	65.7 	65.3 	65.0 	65.1 	64.4 	64.2 	63.9 	63.7 	63.5 	62.7 	62.9 	62.1 	62.1 	61.9 	61.6 	61.0 	60.9 	60.7 	60.2 	60.3 	60.6 	59.7 	59.9 	59.4 	59.4 	59.9 	59.4 	59.4 	59.1 	58.8 	58.6 	58.5 	58.7 	58.0 	58.0 	58.0 	57.4 	57.1 	56.3 	56.2 	55.9 	56.1 	56.6 	55.9 	56.7 	55.5 	56.3 	56.0 	55.7 	56.4 	56.3 	56.2 	56.4 	56.4 	56.2 
YR,M,Y16,LU	: 	64.3 	64.8 	64.6 	64.4 b	64.6 	64.5 	63.7 	64.3 	63.4 b	63.0 	62.3 	62.5 	62.4 	61.0 	61.2 	61.0 	60.4 	59.4 	59.5 	59.9 	59.2 	59.1 	58.3 	58.4 	57.9 	57.7 	57.7 	57.2 	56.9 	57.0 	57.2 	56.3 	55.9 	55.7 	55.7 	55.5 	54.9 	55.2 	54.3 	54.6 	55.0 	54.6 	53.7 	53.8 	53.1 	52.8 	53.2 	53.0 	53.3 	52.7 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 
YR,F,Y17,MT	: 	67.9 	68.5 	68.0 	68.1 	68.0 	67.7 	67.8 	67.7 	66.6 	66.8 	67.1 	66.4 	66.0 	66.0 	65.4 	65.1 	64.9 	64.1 	64.8 	64.6 b	64.1 	63.5 	64.0 	64.3 	64.1 	63.4 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	57.6 	57.0 	57.6 	56.5 	57.8 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 
YR,M,Y66,TR	: 	: 	15.9 	15.9 	15.5 	15.4 	15.5 	15.5 	15.6 	15.3 	14.9 	14.9 	14.5 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 
YR,F,Y7,EA19	: 	77.4 p	78.2 bep	77.9 ep	77.8 b	77.9 	77.5 b	78.0 b	77.6 	77.3 b	77.4 b	77.2 	77.0 	76.8 	76.7 	76.5 	76.0 	76.0 	75.3 	75.4 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 	: 
