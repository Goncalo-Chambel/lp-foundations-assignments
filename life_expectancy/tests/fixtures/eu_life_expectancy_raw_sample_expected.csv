unit,sex,age,region,year,value
YR,T,Y_GE85,PT,2020,6.2
YR,T,Y16,PT,2020,65.4
YR,T,Y_GE85,PT,2019,6.7
YR,T,Y16,PT,2019,66.2
YR,T,Y_GE85,PT,2018,6.5
YR,T,Y16,PT,2018,65.9
YR,T,Y_GE85,PT,2017,6.5
YR,T,Y16,PT,2017,65.9
YR,T,Y_GE85,PT,2016,6.4
YR,T,Y16,PT,2016,65.7
YR,T,Y_GE85,PT,2015,6.3
YR,T,Y16,PT,2015,65.7
YR,T,Y_GE85,PT,2014,6.5
YR,T,Y16,PT,2014,65.7
YR,T,Y_GE85,PT,2013,6.4
YR,T,Y16,PT,2013,65.3
YR,T,Y_GE85,PT,2012,6.2
YR,T,Y16,PT,2012,65.0
YR,T,Y_GE85,PT,2011,6.7
YR,T,Y16,PT,2011,65.1
YR,T,Y_GE85,PT,2010,6.1
YR,T,Y16,PT,2010,64.4
YR,T,Y_GE85,PT,2009,6.1
YR,T,Y16,PT,2009,64.2
YR,T,Y_GE85,PT,2008,6.0
YR,T,Y16,PT,2008,63.9
YR,T,Y_GE85,PT,2007,6.0
YR,T,Y16,PT,2007,63.7
YR,T,Y_GE85,PT,2006,6.1
YR,T,Y16,PT,2006,63.5
YR,T,Y_GE85,PT,2005,5.6
YR,T,Y16,PT,2005,62.7
YR,T,Y_GE85,PT,2004,6.0
YR,T,Y16,PT,2004,62.9
YR,T,Y_GE85,PT,2003,5.4
YR,T,Y16,PT,2003,62.1
YR,T,Y_GE85,PT,2002,5.7
YR,T,Y16,PT,2002,62.1
YR,T,Y_GE85,PT,2001,5.8
YR,T,Y16,PT,2001,61.9
YR,T,Y_GE85,PT,2000,5.6
YR,T,Y16,PT,2000,61.6
YR,T,Y_GE85,PT,1999,5.1
YR,T,Y16,PT,1999,61.0
YR,T,Y_GE85,PT,1998,5.1
YR,T,Y16,PT,1998,60.9
YR,T,Y_GE85,PT,1997,5.1
YR,T,Y16,PT,1997,60.7
YR,T,Y_GE85,PT,1996,4.8
YR,T,Y16,PT,1996,60.2
YR,T,Y_GE85,PT,1995,4.8
YR,T,Y16,PT,1995,60.3
YR,T,Y_GE85,PT,1994,4.9
YR,T,Y16,PT,1994,60.6
YR,T,Y_GE85,PT,1993,4.5
YR,T,Y16,PT,1993,59.7
YR,T,Y_GE85,PT,1992,4.9
YR,T,Y16,PT,1992,59.9
YR,T,Y_GE85,PT,1991,4.6
YR,T,Y16,PT,1991,59.4
YR,T,Y_GE85,PT,1990,4.5
YR,T,Y16,PT,1990,59.4
YR,T,Y_GE85,PT,1989,4.8
YR,T,Y16,PT,1989,59.9
YR,T,Y_GE85,PT,1988,4.5
YR,T,Y16,PT,1988,59.4
YR,T,Y_GE85,PT,1987,4.5
YR,T,Y16,PT,1987,59.4
YR,T,Y_GE85,PT,1986,4.4
YR,T,Y16,PT,1986,59.1
YR,T,Y_GE85,PT,1985,4.1
YR,T,Y16,PT,1985,58.8
YR,T,Y_GE85,PT,1984,4.1
YR,T,Y16,PT,1984,58.6
YR,T,Y_GE85,PT,1983,4.1
YR,T,Y16,PT,1983,58.5
YR,T,Y_GE85,PT,1982,4.5
YR,T,Y16,PT,1982,58.7
YR,T,Y_GE85,PT,1981,4.4
YR,T,Y16,PT,1981,58.0
YR,T,Y_GE85,PT,1980,4.2
YR,T,Y16,PT,1980,58.0
YR,T,Y_GE85,PT,1979,4.1
YR,T,Y16,PT,1979,58.0
YR,T,Y_GE85,PT,1978,3.7
YR,T,Y16,PT,1978,57.4
YR,T,Y_GE85,PT,1977,3.9
YR,T,Y16,PT,1977,57.1
YR,T,Y_GE85,PT,1976,3.6
YR,T,Y16,PT,1976,56.3
YR,T,Y_GE85,PT,1975,3.7
YR,T,Y16,PT,1975,56.2
YR,T,Y_GE85,PT,1974,3.5
YR,T,Y16,PT,1974,55.9
YR,T,Y_GE85,PT,1973,3.9
YR,T,Y16,PT,1973,56.1
YR,T,Y_GE85,PT,1972,4.2
YR,T,Y16,PT,1972,56.6
YR,T,Y_GE85,PT,1971,4.0
YR,T,Y16,PT,1971,55.9
YR,T,Y_GE85,PT,1970,3.9
YR,T,Y16,PT,1970,56.7
YR,T,Y_GE85,PT,1969,3.1
YR,T,Y16,PT,1969,55.5
YR,T,Y_GE85,PT,1968,3.3
YR,T,Y16,PT,1968,56.3
YR,T,Y_GE85,PT,1967,3.3
YR,T,Y16,PT,1967,56.0
YR,T,Y_GE85,PT,1966,3.4
YR,T,Y16,PT,1966,55.7
YR,T,Y_GE85,PT,1965,3.7
YR,T,Y16,PT,1965,56.4
YR,T,Y_GE85,PT,1964,3.9
YR,T,Y16,PT,1964,56.3
YR,T,Y_GE85,PT,1963,4.1
YR,T,Y16,PT,1963,56.2
YR,T,Y_GE85,PT,1962,4.7
YR,T,Y16,PT,1962,56.4
YR,T,Y_GE85,PT,1961,4.8
YR,T,Y16,PT,1961,56.4
YR,T,Y_GE85,PT,1960,4.2
YR,T,Y16,PT,1960,56.2
